docs/sources/tempo/introduction/solutions-with-traces/traces-diagnose-errors.md

```diff
@@ -0,0 +1,83 @@
+---
+description: "Learn how tracing data can help you understand application insights and performance as well as triaging issues in your services and applications."
+keywords:
+  - Grafana
+  - traces
+  - tracing
+title: Diagnose errors with traces
+menuTitle: Diagnose errors with traces
+weight: 400
+---
+
+# Diagnose errors with traces
+
+Traces allow you to quickly diagnose errors in your application, ensuring that you can perform Root Cause Analysis (RCA) on request failures.
+Trace visualizations help you determine the spans in which errors occur, along with the context behind those errors, leading to a lower mean time to repair (MTTR).
+
+## Meet Handy Site Corp
+
+Handy Site Corp, a fake website company, runs an ecommerce application that includes user authentication, a product catalog, order management, payment processing, and other services.
+
+Handy Site’s operations team receives several alerts for their error SLO for monitored endpoints in their services. Using their Grafana dashboards, they notice that there are several issues. The dashboard provides the percentages of errors:
+
+![Dashboard showing errors in services](/media/docs/tempo/intro/traces-error-SLO.png)
+
+More than 5% of the requests from users are resulting in an error across several endpoints, such as `/beholder`, `/owlbear`, and `/illithid`, which causes a degradation in performance and usability.
```

Comment:
Fine to leave it for now but it feels contrived that our e-commerce application has endpoints like "beholder" "owlbear" and "illithid".

Reply:
Agreed that it does seem a little odd. We can iterate on the page and generate new screenshots to replace the ones we have.